src/platform/windows.rs

```diff
@@ -19,6 +19,8 @@ pub type HMENU = isize;
 pub type HMONITOR = isize;
 /// Instance Handle type used by Win32 API
 pub type HINSTANCE = isize;
+/// Message queue type used by Win32 API.
+pub use windows_sys::Win32::UI::WindowsAndMessaging::MSG;
```

Comment:
I guess `*mut c_void` is better. The closure should be able to mutate the `MSG` passed to it as seen in the example.

Reply:
> The closure should be able to mutate the MSG passed to it as seen in the example.

Is there a reason for using a `*mut` pointer instead of `*const`? I currently don't see a reason for being able to mutate the message as the user can also just create a local copy.

Reply:
I meant that as an argument against defining winit's version of `MSG` because I thought `(Translate|Dispatch)Message` functions mutates the passed `MSG` but after checking the docs, it doesn't so both options are viable. I am leaning towards using `*const c_void` but let me know what you prefer.

Reply:
`*const c_void` is fine for me